docs/source/core_concepts/glossary.rst

```diff
@@ -215,6 +215,39 @@ An ordered set of `commands <#command>`__, which is applied to the ledger atomic
 Any nonvalid command within a transaction leads to rejection of the whole
 transaction during the validation process.
 
+Transaction structure
+---------------------
+
+**Payload** stores all transaction fields, except signatures:
+
+    - Time of creation (unix time, in milliseconds)
+    - Account ID of transaction creator (username@domain)
```

Comment:
Please state here that transaction payload also includes "Quorum" field (indicates required number of signatures).